### PR TITLE
Add tool for getting idToken from customToken.

### LINKED ISSUE #70: Implement authentication tool for testing
Currently, only requests with valid `idToken` are accepted so it's quite complicated to call the endpoint manually when testing.  
Create a tool which will take:

* `customToken`
* project API key

and return valid `idToken` suitable for calling the endpoints.

### PR DESCRIPTION
**Pull request by dizider**
Closes #70

### CODE REVIEW

**Review by jendakol — APPROVED** [review]
Please fix the package name (IMHO should be `auth`).

**Review by jendakol — APPROVED** [review]

File: tools/auth/get_id_token.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+	"log"
+	"net/http"
+	"os"
+)
+
+// Key for verify custom token must be saved in PROJECTAPIKEY environment variable
+// example: PROJECTAPIKEY=<projectApiKey> go run get_id_token.go <customToken>
+
+// getIDToken verify customToken and return idToken
+func getIDToken(customToken string, projectAPIKey string) interface{} {
```
Comment (marekaf) [inline tools/auth/get_id_token.go]: wouldn't it be better to return string here?:)

File: tools/auth/get_id_token.go [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+	var result map[string]interface{}
+	err = json.NewDecoder(resp.Body).Decode(&result)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	idToken := result["idToken"]
+	if idToken == nil {
+		log.Fatalln("Invalid projectAPIKey or customToken.")
```
Comment (marekaf) [inline tools/auth/get_id_token.go]: I'd rather return empty interface (or empty string, see the comment above) after Fatalf, just to be sure when somebody changes this from Fatalf to Errorf that the logic does not continue.

File: tools/auth/get_id_token.go [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+func main() {
+	numberOfArguments := len(os.Args)
+
+	if numberOfArguments != 2 {
+		log.Fatalln("Arguments mismatch error. Provide only the custom token as a single argument.")
+	}
+
+	customToken := os.Args[1]
```
Comment (marekaf) [inline tools/auth/get_id_token.go]: might more sense to use flag or cobra libraries to handle the cli flags/args, it is more error-prone and more readable imho
simple flag parsing here https://github.com/google/exposure-notifications-server/blob/main/tools/exposure-client/main.go

**Review by marekaf — APPROVED** [review]
some minor changes requested but otherwise lgtm!

**Pull request merged**